style.css

```diff
@@ -303,3 +333,79 @@ h3 {
   align-items: center;
   justify-content: space-between;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 375px;
+  margin-top: 20%;
+  height: 700px;
+}
+
+.contact-form .description {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 92%;
+  height: 180px;
+  margin-left: 14px;
+  padding: 4px;
+  text-align: center;
+}
+
+.about-form h1 {
+  color: #9390ff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 94%;
+  height: 360px;
+  margin-bottom: 10%;
+  margin-top: 10px;
+  margin-left: 12px;
+  text-align: center;
+}
+
+form input {
+  height: 50px;
+  margin: 12px;
+}
+
+textarea {
+  margin: 12px;
+  min-height: 148px;
+  width: 327px;
+}
+
```

Comment:
- [ ] One of the requirements says that you should stick to the design as much as you can. Kindly your inputs and the textarea miss padding around (`padding: 15px 16px`). check the difference here 👇  👇 

your design | the template
---|--
![image](https://user-images.githubusercontent.com/65068771/150121490-c5cbdb9e-ba37-4dd0-b71e-2ab21327abe1.png) | ![image](https://user-images.githubusercontent.com/65068771/150121641-2260a7b3-1f7f-4158-bec3-92359b47fbc0.png)